spec/models/category_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Please add the test to the models
You can check relations and validations using `shoulda-matchers`

Reply:
i'll do it in next step

Reply:
You can do it in this PR. I think it's more logical
Because you add models now, and you must be sure that all works fine